filter_config_example.txt

```diff
@@ -1,7 +1,7 @@
-flux_lim=1e-3
+flux_lim=5e-4
 SNR=0
 ratio_frac=0
 reject_blends=True
 flags=False
 psf_tol=1.5
-resid_tol=3
\ No newline at end of file
+resid_tol=3
```

Comment:
Should the file be renamed something like 'filter_config_MeerKAT.txt' since I presume you are setting these parameters specific to the MeerKAT continuum QA?

Reply:
Done